atomata/capstone_WebClient
# Action list reorder

### Summary 
- Added the drag and drop functionality to the Action Sequence List.
- Can reorder the buttons on Action Sequence List component. 

### Details
- Added react-beautiful-dnd react library using yarn to be able to make the drag and drop feature. 
- The list in the Action Sequence List can be reordered via drag and drop.
- The elements can be dragged across other components but only locks the position when dropped in Action Sequence List component.
- Jira issue ([EDEX-338](https://atomata.atlassian.net/browse/EDEX-338))

### How to test?
1. Run development side and click "EVIL-CYLINDER" button on the start up page. 
2. Once it loads up, click the "Toggle Overlay" button to see the react components. 
3. Click "WOBBLE-SPHERE" button from the Apparatus List component.
4. Click the "+" button beside the actions in the Action List component to add actions in the  Action Sequence List component.
5. Once the buttons shows up in  Action Sequence List component, drag & drop to reorder and test the feature. 
6. Test drag & drop again by removing the buttons from Action Sequence List component and re adding as well.

### Checks
